tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/guide/guide.js

```diff
@@ -483,6 +484,7 @@ define(['jquery', 'lodash', 'log', 'enjoyhint', 'designViewUtils', 'workspace',
                                 clearInterval(interval);
                             }
                         }, 1000)
+                        browserStorage.put("isWidgetFromTourGuide", false);
```

Comment:
@BuddhiWathsala there can be situations where during the step user can close the guide then browser storage will have this value infenetely. Check whether we can add this to close of the guide and also remove it when editor starts executing if exist